runtime/polkadot/src/lib.rs

```diff
@@ -112,11 +112,6 @@ pub struct BaseFilter;
 impl Filter<Call> for BaseFilter {
 	fn filter(call: &Call) -> bool {
```

Comment:
We can remove all the logic here right?

Reply:
ah indeed. do you prefer so or do you prefer keeping the code like this for next time we add a filtered out call ?

Reply:
tbh doesn't matter to me. We can figure it out later i guess